e2e-tests/IIoTPlatform-E2E-Tests/TestHelper.cs

```diff
@@ -667,6 +667,112 @@ private static PrivateKeyFile GetPrivateSshKey(IIoTPlatformTestContext context)
             }
         }
 
+        /// <summary>
+        /// Equivalent to GetSetOfUniqueNodesAsync
+        /// </summary>
+        /// <param name="context">Shared Context for E2E testing Industrial IoT Platform</param>
+        /// <param name="endpointId">Id of the endpoint as returned by <see cref="Registry_GetEndpoints(IIoTPlatformTestContext)"/></param>
+        /// <param name="nodeId">Id of the parent node or null to browse the root node</param>
+        /// <param name="continuationToken"></param>
+        /// <returns></returns>
+        public static async Task<List<(string NodeId, string NodeClass, bool Children)>> Twin_GetBrowseEndpoint(
```

Comment:
Please add `Async` suffix to the method name and add a `CancellationToken` parameter.